dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/JSSEProviderFactory.java

```diff
@@ -317,101 +315,111 @@ public static void initializeFips() throws Exception {
         if (TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled())
             Tr.entry(tc, "initializeFips");
 
-        if (!fipsInitialized) {
-            int ibmjcefips_position = 0;
-            int ibmjceplusfips_position = 0;
-            Provider[] provider_list = null;
-            Provider ibmjcefips = null;
-            Provider ibmjceplusfips = null;
-            Provider sun = null;
-
-            try {
-                System.setProperty("com.ibm.jsse2.JSSEFIPS", "true");
-                provider_list = Security.getProviders();
+//        if (!fipsInitialized) {
```

Comment:
Why is this logic no longer needed?  Won't it cause behavior change and break customers?

Reply:
I don't agree with this being marked resolved.  You can consider adding something to it to if 140-2 is enabled, but I believe that this logic still needs to be here for customers who did not modify their security providers to add the FIPS providers.  Removing this could break customers that are currently running and move up to a newer version of Liberty.  

If customers using 140-3 are required to update their security providers for FIPS that seems like we are doing them a disservice to make 140-3 harder to configure than 140-2 where we were adding in the providers automatically if they configured FIPS.

Reply:
As per my previous statements marking this resolved.

Reply:
Ok... you convince me!! I will add additional guard for not 140_3  - (!fipsInitialized && !CryptoUtils.isFips140_3Enabled()). 
I will have to revisit it for Semeru 17.

Reply:
As I said I was wrong so you can put it back.